media-libs/opencv/opencv-3.4.0.recipe

```diff
@@ -12,8 +12,9 @@ LICENSE="BSD (3-clause)"
 REVISION="1"
 SOURCE_URI="https://github.com/opencv/opencv/archive/$portVersion.tar.gz"
 SOURCE_FILENAME="opencv-$portVersion.tar.gz"
-CHECKSUM_SHA256="5dca3bb0d661af311e25a72b04a7e4c22c47c1aa86eb73e70063cd378a2aa6ee"
+CHECKSUM_SHA256="678cc3d2d1b3464b512b084a8cca1fad7de207c7abdf2caa1fed636c13e916da"
 SOURCE_DIR="opencv-$portVersion"
```

Comment:
Not needed, as it's the default value.
```diff
-SOURCE_DIR="opencv-$portVersion"
```

BTW, given that there are lots of "3.4.0", you might wish to make these neutral changes:
```diff
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
-	opencv$secondaryArchSuffix = $portVersion compat >= 3
+	opencv$secondaryArchSuffix = $portVersionCompat
```
and replace each `3.4.0 compat >= 3` by `$libVersionCompat`.